core/src/main/java/io/questdb/cairo/sql/DataFrameCursor.java

```diff
@@ -78,7 +82,7 @@ public interface DataFrameCursor extends Closeable, SymbolTableSource {
      * @return data frame and position (lo) of given rowCount (according to cursor order).
      * @throws io.questdb.cairo.DataUnavailableException when the queried partition is in cold storage
      */
-    default @Nullable DataFrame skipTo(long rowCount) {
+    default @Nullable DataFrame skipTo(long rowCount, long[] skippedCount) {
```

Comment:
this is highlighy ambiguous use of function argument without javadoc. I realise this is to return both DataFrame instance and the skipped count, but it has to be documented

Reply:
perhaps even rename the args